apps/okreads-e2e/src/specs/reading-list.spec.ts

```diff
@@ -17,4 +18,56 @@ describe('When: I use the reading list feature', () => {
       )
     );
   });
+
+  it('Then: I should able to add in my reading list', async () => {
```

Comment:
Provide proper description:
`'Then: I should be able to add book in my reading list'`